ChustaSoft.Tools.Authorization/Configuration/ConfigurationHelper.cs

```diff
@@ -34,11 +35,41 @@ public static IdentityBuilder RegisterAuthorization(this IServiceCollection serv
             where TRole : Role, new()
         {
             var authSettings = GetSettingsFromConfiguration(configuration, authSectionName);
-
             return services.RegisterAuthorization<TUser, TRole>(privateKey, authSettings);
         }
 
-        public static IdentityBuilder RegisterAuthorization(this IServiceCollection services, string privateKey, Action<IAuthorizationSettingsBuilder> settingsBuildingAction) 
+        public static void RegisterExternalAuthentication(this IServiceCollection services, IConfiguration configuration)
```

Comment:
As said before, this should be automatically registered once the settings are loaded, regardless of whether it is through the appSettings file or the builder.

Reply:
Logic for external authentication registering has been extracted, and method has been overloaded to allow accept IConfiguration settings or authorization settings builder

Reply:
It is quite close to be as suggested, now this logic should be moved to the main method (RegisterAuthorization) properly loading the settings DTO and doing this logic in a private method.

The idea for this is to have a single line configuration in the Startup. Multiple configuration lines could create confussion and errors for future implementations. Single line eases the set up.